Cloudflare R2 Storage (S3).cyberduckprofile

```diff
@@ -23,6 +23,11 @@
             <string>weur</string>
             <string>eeur</string>
             <string>apac</string>
+            <string>afr</string>

```

Comment:
Please refer me to the documentatino of these regions. I cannot find them in [^1].

[^1]: https://developers.cloudflare.com/r2/reference/data-location/#available-hints